### ISSUE TITLE
User not logging out completely

### ISSUE THREAD

**Issue by Shakroglof** [filebrowser/filebrowser#3627]
**Description**
I'm logged on filebrowser, with 2 tabs opened on the same browser. I log out on one tab.
I can still act on files (for example create a new file) on the other tab until I refresh the page or close the tab

**Expected behaviour**
After loging out of the first tab, any action should be forbidden on the other tab

**Comment by JustinWebDev** [comment]
let's keep the dream alive ...

**Comment by Shakroglof** [comment]
The lack of answer to this security bug is concerning. I installed another app instead.

**Comment by JustinWebDev** [comment]
Yeah, I'm surprised how the devs seem to not find security issues important.

**Comment by JustinWebDev** [comment]
No!  Don't do it!
